plutus-core/plutus-core/src/PlutusCore/Parser/Builtin.hs

```diff
@@ -90,15 +90,7 @@ conArray uniA = Vector.fromList <$> conList uniA
 
 -- | Parser for values.
 conValue :: Parser PLC.Value
-conValue =
-  Value.fromList
-  <$> conList
-        (DefaultUniPair
-          DefaultUniByteString
-          (DefaultUniList
-            (DefaultUniPair DefaultUniByteString DefaultUniInteger)
-          )
-        )
+conValue = Value.fromList <$> conList knownUni
```

Comment:
Can you explain this change? I remember I wasn't sure about the former implementation, so I'd like to understand how this all works.